packages/xo-web/src/common/intl/messages.js

```diff
@@ -2261,6 +2261,8 @@ const messages = {
     'Are you sure you want to destroy {n, number} prox{n, plural, one {y} other {ies}}?',
   destroyProxies: 'Destroy proxy(ies)',
   deployProxy: 'Deploy a proxy',
+  reDeployProxy: 'Re-deploy the proxy',
+  reDeployProxyWarning: 'This action will destroy the old appliance',
```

Comment:
It will be confusing, there have two things, proxy and `XOA` proxy. Redeploying the proxy will not destroy the proxy but it'll destroy the `XOA` proxy.

Reply:
Then call it a "proxy VM", maybe? From a user point of view, XOA runs XO and that's it.

Reply:
`proxy VM` is a good name :+1: